duckduckgo/Android
# New header design: icons, app colors and edit text mode

Task/Issue URL: 
Update icons: https://app.asana.com/0/72649045549333/608322930507367
Update color pallette: https://app.asana.com/0/72649045549333/608322930507366
Add editing design to edit text: https://app.asana.com/0/72649045549333/613439415394753 

Tech Design URL: N/A

**Description**:
Adds new header design

**Steps to test this PR**:
Run the app and check that
• The icons and colors are updated
• Tabs icon is on main page
• The edit text mode hides other icons and shows new background when in edit mode

NOTE: There is a separate task to add a "count" to the tabs button.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
